examples/experimental/i3s-17-and-debug/tile-layer/tile-layer.js

```diff
@@ -35,6 +35,11 @@ export default class TileLayer extends Tile3DLayer {
     if (!timeline || !viewports || !tileset3d) {
       return;
     }
+
+    if (this.internalState.component.props.stopLoading) {
+      return;
+    }
+
     const frameNumber = tileset3d.update(viewports);
```

Comment:
The best solution would be to have a system where tileset3d.update took a list of viewports.

Currently if we have two different viewports that actually are supposed to load, we retraverse the tileset with different parameters.

deck should pass the viewport list through.

Reply:
`viewports` is a list of viewports